MdePkg/MdePkg.dec

```diff
@@ -2252,6 +2257,11 @@
   # @Prompt Speculation Barrier Type.
   gEfiMdePkgTokenSpaceGuid.PcdSpeculationBarrierType|0x01|UINT8|0x30001018
 
+  ## MU_CHANGE START: Add Stack Cookie Exception Vector

```

Comment:
nit: different MU_CHANGE tag then the rest of the PR

Reply:
Going to leave, we don't seem to have a standard beyond MU_CHANGE